### DISCUSSION TITLE
How do we delete a file from filesystem in express.js

### DISCUSSION THREAD

**Discussion by boringlysol** [rok0705/Image_upload#3] [Q&A] [answered]
title says all..

**Comment by rok0705** [comment, marked as answer]
1. import fs library.
const { fs } = require("fs");

2. import promisify.
const { promisify } = require("util");

3. wrap unlink() call with Promise. This is required because unlink() is asynchronous call.
const fileUnlink = promisify(fs.unlink);

4. use fileUnlink

await fileUnlink(`./database/${image}.jpeg`);
res.json("a file delete successfully.");
